app/assets/javascripts/analytics-track-form.js

```diff
@@ -7,27 +7,21 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
 (function (global, GOVUK) {
   'use strict'
 
-  var $ = global.jQuery
-
   GOVUK.Modules.TrackForm = function () {
     this.start = function (element) {
-      track(element)
-    }
-
-    function track (element) {
-      element.on('submit', function (event) {
+      element[0].addEventListener('submit', function(event) {
         var $checkedOption, questionValue
-        var $submittedForm = $(event.target)
-        var $checkedOptions = $submittedForm.find('input:checked')
-        var trackCategory = $submittedForm.data('track-category')
+        var $submittedForm = event.target
+        var $checkedOptions = $submittedForm.querySelectorAll('input:checked')
+        var trackCategory = $submittedForm.getAttribute('data-track-category')
 
         if ($checkedOptions.length) {
-          $checkedOptions.each(function (index) {
-            $checkedOption = $(this)
-            var trackAction = $checkedOption.data('track-action')
-            var checkedOptionId = $checkedOption.attr('id')
-            var checkedOptionLabel = $submittedForm.find('label[for="' + checkedOptionId + '"]').text().trim()
-            questionValue = checkedOptionLabel.length ? checkedOptionLabel : $checkedOption.val()
+          for (var i = 0; i < $checkedOptions.length; i++) {
+            $checkedOption = $checkedOptions[i]
+            var trackAction = $checkedOption.getAttribute('data-track-action')
+            var checkedOptionId = $checkedOption.getAttribute('id')
+            var checkedOptionLabel = $submittedForm.querySelector('label[for="' + checkedOptionId + '"]').innerText.trim()
```

Comment:
If a `label[for="checkedOptionId"]` does not exist,  this line will fail with a `TypeError: Cannot read property 'innerText' of null`. 

JQuery fails more gracefully in that if an element does not exist it still returns an empty jQuery object, so then as a result `text()` would return an empty string.

Reply:
Thanks, good spot. Have pushed an extra commit to allow and test for this possibility, if you're able to review again?